amazon-secrets-manager-properties-provider/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,68 @@
+
+For information about compatibility and fixed issues, refer to the Amazon Secrets Manager Properties Provider Release Notes.
+
+== Before You Begin
+
+To use this properties provider, you must be familiar with:
+
+* AWS Secrets Manager
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* How to create a Mule app using Anypoint Studio
+
+Before creating an app, you must have:
+
+* Mule Enterprise Edition
+* Credentials to access the AWS Secrets
+* Anypoint Platform
+* Anypoint Studio version 7.3.5 or later
+
+== Common Use Cases for the Properties Provider
+
+With AWS Secrets Manager Properties Provider, you can:
+
+* Configure values from AWS secrets manager for Mule applications.
+* Retrieve secrets for SaaS applications as part of the trigger or action.
+* Specify the name of the secret to retrieve.
+* Authenticate to AWS using the available options from the AWS SDK IAM, default provider chain or Basic Authentication.
+* Specify the AWS server or region to connect to the appropriate server for your business unit.
+* Be notified about errors while connecting to AWS.
+* Be notified about errors while retrieving secretes from AWS.
+
+== Connection Types
+
+Connectors can use the Amazon Secrets Manager Properties Provider in their configurations. For example, in the *Global Element Properties* window of Anypoint Studio, you can configure the following fields for the *AWS Secrets Manager Properties Override Config*:
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Name | Type | Description | Default Value | Required
+| AWS Secrets Manager Region a| String |  AWS Secrets Manager region, for example `us-east-2`. |  | x
+| AWS Access Key a| String |  AWS access key ID. |  | 
+| AWS Secret Key a | String |  AWS secret key ID. |  | 
+|Role ARN a| String | The Role ARN unique identifies a role to gain cross account access. |  |
+|===
+
+image::amazon-secrets-manager-properties.png[Example of Amazon Secrets Manager Properties Provider in Global Element Properties window of Anypoint Studio]
+
+In the *Configuration XML* editor, the configuration looks like this:
+
+[source,xml,linenums]
+----
+<aws-secrets-manager-properties-override:config name="AWS_Secrets_Manager_Properties_Override_Config" doc:name="AWS Secrets Manager Properties Override Config" >
+		<aws-secrets-manager-properties-override:basic-connection region="us-east-1" />
+		<aws-secrets-manager-properties-override:role-connection roleARN="arn:aws:iam::480410955647:role/AmazonSSMRoleForAutomationAssumeQuickSetup" />
+	</aws-secrets-manager-properties-override:config>
+----
+
+If you do not provide credentials, the default AWS connection follows the https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials-chain.html[default credentials provider chain - AWS SDK for java 2.x^]
+
+== See Also
+
```

Comment:
an optional suggestion: standardize the terminology of the name, like replace AWS Secrets Manager with Amazon Secrets Manager